src/hotspot/share/gc/g1/g1CardSetContainers.hpp

```diff
@@ -178,6 +178,8 @@ class G1CardSetContainer {
     _next = next;
   }
 
+  ~G1CardSetContainer() = default;
```

Comment:
I'd prefer the destructor declaration be close to the constructor, i.e. cluster the special function declarations.